pkg/agent/flowexporter/connections/conntrack_windows.go

```diff
@@ -23,33 +23,15 @@ import (
 	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
 )
 
-var _ ConnTrackDumper = new(connTrackDumper)
-
-type connTrackDumper struct {
-	nodeConfig  *config.NodeConfig
-	serviceCIDR *net.IPNet
-	connTrack   ConnTrackInterfacer
-}
-
-func NewConnTrackDumper(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet, conntrack ConnTrackInterfacer) *connTrackDumper {
-	return &connTrackDumper{
-		nodeConfig,
-		serviceCIDR,
-		conntrack,
-	}
-}
-
-// TODO: These will be defined when polling from ovs-dpctl dump conntrack is supported for windows.
-var _ ConnTrackInterfacer = new(connTrackSystem)
-
-type ConnTrackInterfacer interface{}
+// TODO:  Implement ConnTrackDumper for windows.
+var _ ConnTrackDumper = new(connTrackSystem)
 
 type connTrackSystem struct{}
 
-func NewConnTrackInterfacer() *connTrackSystem {
-	return &connTrackSystem{}
+func NewConnTrackSystem(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet) *connTrackSystem {
+	return nil
 }
 
-func (cp *connTrackDumper) DumpFlows(zoneFilter uint16) ([]*flowexporter.Connection, error) {
+func (ct *connTrackSystem) DumpFlows(zoneFilter uint16) ([]*flowexporter.Connection, error) {
 	return nil, nil
```

Comment:
please check, but my understanding is that on Windows we just use a conntrack implementation provided by OVS (https://github.com/openvswitch/ovs/tree/master/datapath-windows/ovsext). I believe that based on this observation, your implementation of `ConnTrackDumper` which uses `ovs-appctl` (for OVS netdev datapath) will also work on Windows (and may be our only option?).

obviously we can / should tackle this in a future PR, but in the mean time do you think you could rename the `connTrackSystem` struct to something else? I am not sure it makes sense to have the same name for the 2 implementations (Linux kernel, Windows OVS datapath) even if they are never build together.

Reply:
Made the windows file much simple by just having the constructor function.